### PR TITLE
[AMORO-2742] Update issue link to apache

### PR DESCRIPTION
**Pull request by xleoken**
## Brief change log

Update issue link to apache.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Review by zhongqishang — APPROVED** [review]
LGTM. Thanks for your contribution.

File: .idea/vcs.xml [hunk 1]
```diff
@@ -5,11 +5,11 @@
       <list>
         <IssueNavigationLink>
           <option name="issueRegexp" value="(ARCTIC|AMORO)\-(\d+)" />
-          <option name="linkRegexp" value="https://github.com/NetEase/arctic/issues/$2" />
+          <option name="linkRegexp" value="https://github.com/apache/amoro/issues/$2" />
         </IssueNavigationLink>
         <IssueNavigationLink>
           <option name="issueRegexp" value="#(\d+)" />
-          <option name="linkRegexp" value="https://github.com/NetEase/arctic/pull/$1" />
+          <option name="linkRegexp" value="https://github.com/apache/amoro/pulls/$1" />
```
Comment (zhongqishang) [inline .idea/vcs.xml:12]: Sorry I miss one that `/pull` should not be changed to `/pulls`.

Comment (baiyangtx) [inline .idea/vcs.xml:12]:
```suggestion
          <option name="linkRegexp" value="https://github.com/apache/amoro/pull/$1" />
```

**Review by baiyangtx — CHANGES_REQUESTED** [review]

**Comment by xleoken** [comment]
thanks @baiyangtx @zhongqishang

File: .idea/vcs.xml (hunk 1 above)
Comment (xleoken) [reply .idea/vcs.xml]: updated.

**Review by baiyangtx — APPROVED** [review]
LGTM
